### ISSUE TITLE
Take point value from pricedb instead of points-value

### ISSUE THREAD

**Issue by tbm** [redstreet/fava_miler#3]
You require a `points-value` on the commodity definition.

But since beancount has a `price` directive, wouldn't it make more sense to take the price from there?

**Comment by tbm** [comment]
I can implement this if you agree.

**Comment by redstreet** [comment]
Good idea, but I need to think about this more. Doing so brings up a few issues:
- do we want to be able to have changing prices (I can see arguments for both sides)
- what if the pricedb specifies prices in multiple currencies?
- would specifying it in the pricedb cause unintended conversions (eg: in net worth)

**Comment by tbm** [comment]
These are good points, thanks.

* I believe point values can change (i.e. multiple pricedb entries).  We should just use the latest value.
* Multiple currencies?  I think just use the latest pricedb entry
* Yes, they would show up in net worth. But either they have a value or they don't. Maybe there should be a way to exclude certain commodities from net worth. That's probably a fava-wide feature that might be nice.

**Comment by redstreet** [comment]
> Multiple currencies? I think just use the latest pricedb entry

I don't think this is a good idea. Letting the user specify a currency via the UI would be better. The problem is when a conversion is not available in that currency but available in another.

The current solution side steps all of this by forcing the user to specify a single conversion at the cost of a bit of flexibility.

> Yes, they would show up in net worth. But either they have a value or they don't.

I'm not sure I agree. My points values are estimates, which are between "having a value" and not. I'd rather see my miles separately in net worth than a conversion that's an estimate.

One solution is to use priced, but allow commodity metadata to override it. The UI issue would still need to be solved.

**Comment by tbm** [comment]
> I don't think this is a good idea. Letting the user specify a currency via the UI would be better

Right, this is similar to the discussion at https://github.com/redstreet/fava_investor/issues/32
I'm not sure if Fava's interface for currency selection is available for plugins, but it seems like using the same approach would be best.

> One solution is to use priced, but allow commodity metadata to override it

Right, that sounds sensible to me.

**Comment by redstreet** [comment]
Sure, feel free to send a PR.

The conversion menu is available to extensions. Currencies is available only as a part of a chart.

**Comment by tbm** [comment]
> Sure, feel free to send a PR.

I'll try but I can't promise since I don't know how the conversion menu works

**Comment by redstreet** [comment]
The example that ships with Fava should help.
